### ISSUE TITLE
[pt2e]the Acc and Perf data are not quite normal

### ISSUE THREAD

**Issue by kaileiyx** [intel/torch-xpu-ops#1717]
### 🐛 Describe the bug

The latest nightly PT2E test data as follows:

### Accuracy

| Model | fp32 Acc@1 | fp32 Acc@5 | int8 Acc@1 | int8 Acc@5 | int8/fp32 Acc@1 | int8/fp32 Acc@5 |
| - | - | - | - | - | - | -|
| alexnet | 56.56 | 79.08 | 56.56 | 79.08 | 100.00% | 100.00% |
| mnasnet1_0 | 73.47 | 91.52 | 73.47 | 91.52 | 100.00% | 100.00% |
| mobilenet_v2 | 71.87 | 90.29 | 71.87 | 90.29 | 100.00% | 100.00% |
| mobilenet_v3_large | 74.05 | 91.34 | 74.05 | 91.34 | 100.00% | 100.00% |
| resnet152 | 78.31 | 94.06 | 78.31 | 94.06 | 100.00% | 100.00% |
| resnet18 | 69.76 | 89.08 | 69.76 | 89.08 | 100.00% | 100.00% |
| resnet50 | 76.15 | 92.87 | 76.15 | 92.87 | 100.00% | 100.00% |
| resnext50_32x4d | 77.61 | 93.69 | 77.61 | 93.69 | 100.00% | 100.00% |
| shufflenet_v2_x1_0 | 69.36 | 88.31 | 69.36 | 88.31 | 100.00% | 100.00% |
| squeezenet1_1 | 58.19 | 80.62 | 58.19 | 80.62 | 100.00% | 100.00% |
| vgg16 | 71.59 | 90.39 | 71.59 | 90.39 | 100.00% | 100.00% |

### Performance
| Model | fp32 | int8_ASYMM | int8_SYMM | ASYMM  int8/fp32 | SYMM  int8/fp32 |
| - | - | - | - | - | - |
| alexnet | 10466.00 | 10449.17 | 10483.62 | 1.00 | 1.00 |
| demucs | 109.02 | 111.59 | 111.81 | 1.02 | 1.03 |
| dlrm | 1212.96 | 1203.34 | 1198.77 | 0.99 | 0.99 |
| hf_Albert | 152.50 | 152.30 | 152.40 | 1.00 | 1.00 |
| hf_Bert | 132.25 | 132.03 | 131.58 | 1.00 | 0.99 |
| hf_Bert_large | 50.34 | 51.01 | 51.02 | 1.01 | 1.01 |
| hf_DistilBert | 214.28 | NULL | NULL | NULL | NULL |
| hf_Roberta_base | 63.42 | 63.31 | 63.28 | 1.00 | 1.00 |
| mnasnet1_0 | 7120.06 | 7164.48 | 7168.78 | 1.01 | 1.01 |
| mobilenet_v2 | 6697.73 | 6707.56 | 6794.94 | 1.00 | 1.01 |
| mobilenet_v3_large | 8464.31 | 8822.11 | 8723.69 | 1.04 | 1.03 |
| nvidia_deeprecommender | 42118.51 | 42111.76 | 42131.76 | 1.00 | 1.00 |
| pytorch_CycleGAN_and_pix2pix | 118.58 | 118.96 | 117.85 | 1.00 | 0.99 |
| resnet152 | 636.99 | 642.36 | 641.91 | 1.01 | 1.01 |
| resnet18 | 4200.13 | 4216.85 | 4211.48 | 1.00 | 1.00 |
| resnet50 | 1648.59 | 1662.10 | 1663.01 | 1.01 | 1.01 |
| resnext50_32x4d | 1441.72 | 1450.79 | 1452.97 | 1.01 | 1.01 |
| shufflenet_v2_x1_0 | 15308.53 | 15815.22 | 15861.21 | 1.03 | 1.04 |
| squeezenet1_1 | 10228.33 | 10239.79 | 10258.87 | 1.00 | 1.00 |
| Super_SloMo | 49.29 | 49.52 | 49.54 | 1.00 | 1.00 |
| timm_efficientnet | 4934.27 | 5385.29 | 5379.93 | 1.09 | 1.09 |
| timm_nfnet | 1032.06 | NULL | NULL | NULL | NULL |
| timm_regnet | 590.50 | 599.21 | 598.32 | 1.01 | 1.01 |
| timm_resnest | 2420.70 | 2530.73 | 2530.31 | 1.05 | 1.05 |
| timm_vision_transformer | 1415.58 | 1423.37 | 1420.38 | 1.01 | 1.00 |
| timm_vision_transformer_large | 32.45 | 32.44 | 32.43 | 1.00 | 1.00 |
| timm_vovnet | 1097.27 | 1101.85 | 1101.43 | 1.00 | 1.00 |
| vgg16 | 472.76 | 473.83 | 472.85 | 1.00 | 1.00 |

The acc data are the same. the perf data are almost identical

### Versions

When this situation firest occurred, the pytorch commit ID: dab7e5700392e4e20626de9c367acb76187807f5

**Comment by libohao1201** [comment]
For Windows,  the accuracy is the same as that of Linux. The performance is as follows:
| Model                         | fp32     | int8_ASYMM | int8_SYMM | ASYMM    int8/fp32 | SYMM    int8/fp32 |
|-------------------------------|----------|------------|-----------|--------------------|-------------------|
| alexnet                       | 7259.384 | 7318.32    | 7318.32   | 1.008119           | 1.008119          |
| demucs                        | 28.87733 | 43.40735   | 43.40735  | 1.503163           | 1.503163          |
| dlrm                          | 677663.3 | 679653.7   | 679653.7  | 1.002937           | 1.002937          |
| hf_Albert                     | 95.60574 | 97.67202   | 97.67202  | 1.021612           | 1.021612          |
| hf_Bert                       | 82.65727 | 83.62917   | 83.62917  | 1.011758           | 1.011758          |
| hf_Bert_large                 | 32.48061 | 31.98485   | 31.98485  | 0.984737           | 0.984737          |
| hf_DistilBert                 | 141.9386 |            |           |                    |                   |
| hf_Roberta_base               | 40.20577 | 41.24621   | 41.24621  | 1.025878           | 1.025878          |
| mnasnet1_0                    | 3999.625 | 4201.736   | 4201.736  | 1.050532           | 1.050532          |
| mobilenet_v2                  | 3692.039 | 3962.554   | 3962.554  | 1.07327            | 1.07327           |
| mobilenet_v3_large            | 4799.008 | 4545.81    | 4545.81   | 0.947239           | 0.947239          |
| nvidia_deeprecommender        | 28335.04 | 28950.13   | 28950.13  | 1.021708           | 1.021708          |
| pytorch_CycleGAN_and_pix2pix  | 89.66117 | 91.56461   | 91.56461  | 1.021229           | 1.021229          |
| resnet152                     | 397.9696 | 401.094    | 401.094   | 1.007851           | 1.007851          |
| resnet18                      | 2632.142 | 2643.93    | 2643.93   | 1.004479           | 1.004479          |
| resnet50                      | 1039.128 | 1051.162   | 1051.162  | 1.011581           | 1.011581          |
| resnext50_32x4d               | 875.8681 | 897.4281   | 897.4281  | 1.024615           | 1.024615          |
| shufflenet_v2_x1_0            | 8390.808 | 9055.408   | 9055.408  | 1.079206           | 1.079206          |
| squeezenet1_1                 | 5303.457 | 5302.051   | 5302.051  | 0.999735           | 0.999735          |
| Super_SloMo                   | 30.957   | 31.67275   | 31.67275  | 1.023121           | 1.023121          |
| timm_efficientnet             | 2696.02  | 3018.121   | 3018.121  | 1.119473           | 1.119473          |
| timm_nfnet                    | 686.9439 |            |           |                    |                   |
| timm_regnet                   | 373.1196 | 381.9145   | 381.9145  | 1.023571           | 1.023571          |
| timm_resnest                  | 1499.876 | 1585.473   | 1585.473  | 1.057069           | 1.057069          |
| timm_vision_transformer       | 899.5309 | 906.1472   | 906.1472  | 1.007355           | 1.007355          |
| timm_vision_transformer_large | 20.91415 |            |           |                    |                   |
| timm_vovnet                   | 702.47   | 705.1471   | 705.1471  | 1.003811           | 1.003811          |
| vgg16                         | 302.8258 | 306.1228   | 306.1228  | 1.010888           | 1.010888          |

**Comment by CuiYifeng** [comment]
PT2E workflow was broken by `torch.export.export_for_training` API change. The default value of parameter `strict` was flipped from True to False. Need to explicitly set `strict=True` in test scripts.
